roles/artemis/templates/application-prod.yml.j2

```
@@ -340,9 +340,11 @@ jhipster:
         token-validity-in-seconds-for-remember-me: 2592000  # Token is valid 30 days
 {% endif %}
 
-{% if artemis_CORS_allowed_origins is defined and artemis_CORS_allowed_origins is not none %}
-  cors:
-    allowed-origins: {{ artemis_CORS_allowed_origins }}
+{% if artemis_cors is defined and artemis_cors is not none %}
+  artemis_cors:
+    {% for key, value in cors.items() %}
```

This probably should be artemis_cors instead of cors
```suggestion
    {% for key, value in artemis_cors.items() %}
```

---

Also why do you iterate through the dict and then convert each entry to yaml instead of just converting the entire dict directly to yaml?

---

Another question: I see the point of not hard coding the keys and having the option of the dynamic configuration, but in the `artemis.env.j2` file the keys are hard coded.

Therefore, I would prefer an homogeneous approach and that the keys are hard coded here as well.

---

Well, I reiterated over that because I need a similar approach for something else (Theia configuration). Thus, I came up with a potential approach for a dynamic configuration in the environment file.